docker-compose.e2e.yml

```diff
@@ -0,0 +1,40 @@
+version: "3.9"
+services:
+  firefox:
+    image: selenium/standalone-firefox:${FIREFOX_VERSION:-latest}
```

Comment:
There's a disclaimer in the `playwright/firefox` image:
> These images only work with Moon software (https://aerokube.com/moon/). Official Playwright team images are located here: https://hub.docker.com/_/microsoft-playwright

It looks like the browsers are embedded in the main playwright image though - https://playwright.dev/docs/docker. Less customisable but I think it might be OK for this purpose.

There's a bigger blocker however. It looks like loading Firefox extensions is not supported by playwright. See https://github.com/microsoft/playwright/issues/7297